### PR TITLE
Page Web Options

### PR DESCRIPTION
**Pull request by Rafval18**

### CODE REVIEW

File: .vscode/settings.json
Comment (KaruzG) [inline .vscode/settings.json:1]: Dont push your local vscode configuration please

File: Cosas/A1
Comment (KaruzG) [inline Cosas/A1:1]: What is this??

File: public/css/css/options.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Arial, sans-serif;
```
Comment (KaruzG) [inline public/css/css/options.css]: Not the font we are using

File: public/js/settings.js
Comment (KaruzG) [inline public/js/settings.js:1]: Encapsulate the code

**Review by KaruzG — CHANGES_REQUESTED** [review]

**Comment by KaruzG** [comment]
Also, what? This folders? THis would break things
image

File: Cosas/A1
Comment (Rafval18) [reply Cosas/A1:1]: The web page options! :D

File: .vscode/settings.json
Comment (Rafval18) [reply .vscode/settings.json:1]: ok

File: public/css/css/options.css (hunk 1 above)
Comment (Rafval18) [reply public/css/css/options.css]: It already works more or less correctly

File: public/js/settings.js
Comment (Rafval18) [reply public/js/settings.js:1]: image

Poniendo function, ¿No estaría encapsulado?

File: Cosas/A1
Comment (KaruzG) [reply Cosas/A1:1]: I mean "Cosas/A1" two empty folders

File: public/js/settings.js
Comment (KaruzG) [reply public/js/settings.js:1]: I was talking about this
image

**Comment by KaruzG** [comment]
I fixed the problems

**Review by KaruzG — APPROVED** [review]

**Review by JME469 — APPROVED** [review]
good to go

**Pull request merged**